migration/src/test/java/org/axonframework/migration/AxonJakartaTest.java

```diff
@@ -0,0 +1,79 @@
+package org.axonframework.migration;
+
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class AxonJakartaTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+                .parser(JavaParser.fromJavaVersion()
+                        .logCompilationWarningsAndErrors(true)
+                        .classpath("axon-messaging", "rewrite-spring"))
```

Comment:
Curious: why have `rewrite-spring` here?

Reply:
Because of the (now commented out) chained SpringBoot 3 migration; it also brings in transitive migration recipes to Jakarta / Java 17 / JUnit that you might want to include if applicable. If not it can be removed from here.

Reply:
Since swapped out for `rewrite-migrate-java` to only chain the jakarta migration for now.